cp3pt0-deployment/setup_singleton.sh

```diff
@@ -227,4 +235,16 @@ function get_and_validate_arguments() {
     get_control_namespace
 }
 
+function accept_license() {
```

Comment:
Need to generalize the function because licensing-operator will also have license acceptance, so this will need to be called whenever they finish implementing license acceptance.

Suggest function to look like:
```
accept_license:
  arg1 kind - e.g. certmanager or ibmlicensing
  cr := oc get $kind
  if cr is empty then warning msg and return
  oc patch cr with .spec.license.accept: true
  if error then warning msg
```